### ISSUE TITLE
Alerts policy not found during test

### ISSUE THREAD

**Issue by RubyLi0612** [terraform-providers/terraform-provider-newrelic#11]
### Terraform Version
Terraform v0.9.8

### Affected Resource(s)
newrelic_alert_policy_channel

### Terraform Configuration Files
```hcl
resource "newrelic_alert_policy" "foo" {
  name = "tf-test-%[1]s"
}

resource "newrelic_alert_channel" "foo" {
  name = "tf-test-%[1]s"
	type = "email"
	
	configuration = {
		recipients = "foo@example.com"
		include_json_attachment = "1"
	}
}

resource "newrelic_alert_policy_channel" "foo" {
  policy_id  = "${newrelic_alert_policy.foo.id}"
  channel_id = "${newrelic_alert_channel.foo.id}"
}
```

### Debug Output
[https://gist.github.com/RubyLi0612/5945cc7cf6c658d8b30da09561def4bc](url)

### Expected Behavior
Pass tests under `github.com/terraform-providers/terraform-provider-newrelic/newrelic/`

### Actual Behavior
Test fail for creating new NewRelic alert policy channel:
<img width="1339" alt="screen shot 2017-07-05 at 2 18 11 pm" src="https://user-images.githubusercontent.com/29551647/27885422-dbd8bcb2-618c-11e7-9823-1688b074a19c.png">

This only occur sometimes, while some time the test passes without error.

### Steps to Reproduce
Please list the steps required to reproduce the issue, for example:
1. `cd github.com/terraform-providers/terraform-provider-newrelic/`
2. `make fmt`
3. set NEWRELIC_API_KEY and NEWRELIC_LICENSE_KEY correctly
4. `make testacc`

### Important Factoids
The files are pulled from this repo unchanged. All imported files are also cloned from according repos.

**Comment by kleeadrian** [comment]
I ran into this error as well.  as a temporary workaround, i had to rename my .tfstate file for it to deploy again.

**Comment by ctrombley** [comment]
Addressed in #58.
